keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -810,21 +784,21 @@ spec:
 ### Test Plan
 
 - Add e2e tests for ClusterNetworkPolicy resource
-  - Ensure `Empower`rules are always allowed
+  - Ensure `Empower`rules are excepted
```

Comment:
Ensure `Empower` rules override `Deny` rules and provide exceptions to the `Deny` rules?

Reply:
done